css/news.css

```diff
@@ -0,0 +1,96 @@
+.news {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  padding: 141px 0;
+}
+
+.news__block {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.news__item {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-right: 30px;
+}
+
+.news__item:last-of-type {
+  margin-right: 0;
+}
+
+.news__item-description {
+  display: flex;
+  /*align-items: left;*/
+  justify-content: center;
+  flex-direction: column;
+  width: 370px;
+  height: 249px;
+  box-shadow: 0 0 3px rgba(0, 0, 0, 0.2);
+}
+
+.news__item-title {
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 24px;
+  color: #474747;
+  padding-left: 29px;
+}
+
+.news__item-title:hover {
```

Comment:
you need to change text color, not even you move the cursor to block `title`,
change text color on when you move the cursor anywhere in block `item`

try something like this
`.news__item-title:hover {` => `. news__item:hover news__item-title {`